CautionaryAlertsApi/V1/Gateways/IUhGateway.cs

```diff
@@ -18,8 +19,9 @@ public interface IUhGateway
 
         Task<IEnumerable<CautionaryAlertListItem>> GetCautionaryAlertsByMMHPersonId(Guid personId);
 
-        CautionaryAlert GetCautionaryAlertByAlertId(AlertQueryObject query);
+        PropertyAlertDomain GetCautionaryAlertByAlertId(AlertQueryObject query);
```

Comment:
Better to call this just PropertyAlert and leave "Domain" out as it makes it less descriptive of its responsibility

Reply:
It's a Hackney.Shared cautionary alerts nuget package class - we would likely benefit from a wider discussion before changing that. Plus there's some tech debt surrounding that nuget that needs resolving anyway. I'd argue it's a separate ticket.